website/docs/api/plugins/plugin-content-docs.md

```diff
@@ -34,6 +34,7 @@ Accepted fields:
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | `path` | `string` | `'docs'` | Path to the docs content directory on the file system, relative to site directory. |
+| `versionedDocsPath` | `string` | `undefined` | Path to the versioned docs content directory on the file system, relative to site directory. This is where the CLI command `docusaurus docs:version 1.2.3` will write the files to. The default is the site directory. |
```

Comment:
It was not clear to me at all reading this doc that the default value is not `website/versioned_docs` but rather simply `website` and that it affects where both `versioned_docs` and `versioned_sidebars` will be written.

Wondering if it wouldn't be better to have 2 standalone options to give more flexibility? 🤷‍♂️